tests/workflows/test_uber_lyft.py

```diff
@@ -0,0 +1,97 @@
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pytest
+from dask.distributed import Client
+
+
+@pytest.fixture(scope="module")
+def uber_lyft_cluster(
+    dask_env_variables,
+    cluster_kwargs,
+    github_cluster_tags,
+):
+    with coiled.Cluster(
+        f"uber-lyft-{uuid.uuid4().hex[:8]}",
+        environ=dask_env_variables,
+        tags=github_cluster_tags,
+        **cluster_kwargs["uber_lyft_cluster"],
+    ) as cluster:
+        yield cluster
+
+
+@pytest.fixture
+def uber_lyft_client(
+    uber_lyft_cluster,
+    cluster_kwargs,
+    upload_cluster_dump,
+    benchmark_all,
+):
+    n_workers = cluster_kwargs["uber_lyft_cluster"]["n_workers"]
+    with Client(uber_lyft_cluster) as client:
+        uber_lyft_cluster.scale(n_workers)
+        client.wait_for_workers(n_workers)
+        client.restart()
+        with upload_cluster_dump(client), benchmark_all(client):
+            yield client
+
+
+@pytest.fixture
+def ddf(uber_lyft_client):
+    """NYC taxi Uber/Lyft dataset"""
+    columns = [
+        "hvfhs_license_num",
+        "tips",
+        "base_passenger_fare",
+        "driver_pay",
+        "trip_miles",
+        "trip_time",
+        "shared_request_flag",
+    ]
```

Comment:
Why select just these columns? My sense is most folks will just read the whole dataset in and then select the columns they need afterwards. Over time, once we have a more generic column selection / predicate pushdown / etc., these types of things will automatically improve for users.

Reply:
I'm only calculating a few metrics here and not using all the columns, so it makes more sense to load a subset to decrease the size of data. We would recommend the same for users and their real-world workflows, this is also what @ncclementi does in her [tutorial](https://github.com/coiled/dask-tutorial/blob/main/2-Get_better-at-dask-dataframes.ipynb).